templates/socat.service.j2

```diff
@@ -1,8 +1,10 @@
 [Unit]
 Description=socat tunnel service {{ socat_service_name }}
-After=network.target
+After=network-online.target
+Wants=network-online.target
 
 [Service]
+Type=forking
```

Comment:
I refer to `socat` as a traditional UNIX service and still hope to be able to add the PID file at startup.
https://www.freedesktop.org/software/systemd/man/systemd.service.html#Options

Reply:
yeah but if you read about the `forking`-type - it states it's for services which will fork into background - as far as I know socat usually stays in foreground and can therefore be controlled by systemd directly.

I'm not sure if there are socat options to force it into background - there might be -> maybe we should only add this if the configuration matches a certain (background forking) pattern

part from the forking section:
```
The parent process is expected to exit when start-up is complete and all communication channels are set up.
```

Reply:
I'll merge and switch this back to simple